tests/test_job.py

```diff
@@ -705,6 +705,20 @@ def test_job_tasks_table(hq_env: HqEnv):
     assert worker == "" or worker == "worker1"
 
 
+def test_job_tasks_makespan(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+
+    hq_env.command(["submit", "sleep", "10"])
+    hq_env.command(["job", "cancel", "1"])
+
+    table_1 = hq_env.command(["job", "tasks", "1"])
```

Comment:
1) Let's wait for the job to be cancelled before showing the task table.
2) Check that the makespan is not empty.